src/toil_scripts/adam_pipeline/adam_preprocessing.py

```diff
@@ -354,66 +353,67 @@ def scale_external_spark_cluster(num_samples=1):
         sem.release(delta=-num_samples)
 
 
-def build_parser():
+def generate_config():
+    return textwrap.dedent("""
+        # ADAM Preprocessing Pipeline configuration file
+        # This configuration file is formatted in YAML. Simply write the value (at least one space) after the colon.
+        # Edit the values in this configuration file and then rerun the pipeline
+        # Comments (beginning with #) do not need to be removed. Optional parameters may be left blank.
+        ##############################################################################################################
+        num-nodes: 9              # Optional: Number of nodes to use. Do not set if providing master_ip.
+        master-ip:                # Optional: IP or hostname of host running for Spark master and HDFS namenode.
+                                  # Should be provided instead of num-nodes if pointing at a static (external or
+                                  # standalone) Spark cluster. The special value 'auto' indicates the master of
+                                  # standalone cluster, i.e. one that is managed by the uberscript.
```

Comment:
Might want to clarify this a bit. I assume that `auto` is the _external_ autoscaled Spark cluster? I would avoid the use of the phrase "standalone", since "standalone" in Spark parlance more or less officially refers to a Spark cluster that is running without Mesos or YARN (or etc), and here it just confuses the docs a bit.

I mean, yes, our external cgcloud Spark cluster is a standalone Spark cluster, but the point is more that it is the externally autoscaled cluster.